addons/portal/static/src/interactions/portal_picture.js

```diff
@@ -0,0 +1,80 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+import { _t } from "@web/core/l10n/translation";
+
+export class ProfilePicture extends Interaction {
+    static selector = ".o_portal_picture_card";
+    dynamicContent = {
+        ".o_file_upload": { "t-on-change": this.onFileChange },
+        ".o_portal_profile_pic_edit": { "t-on-click.prevent": this.onEditProfilePicClick },
+        ".o_portal_profile_pic_clear": { "t-on-click.prevent": this.onClearProfilePicClick },
+    };
+
+    setup() {
+        this.notification = this.services.notification;
+        this.fileUploadEl = this.el.querySelector(".o_file_upload");
+        this.avatarImgEl = this.el.querySelector(".o_wportal_avatar_img");
+        this.lastValidFile = this.avatarImgEl.src;
```

Comment:
```suggestion
        this.lastValidImage = this.avatarImgEl.src;
```

This sounds more precious.